setup.py

```diff
@@ -30,6 +30,7 @@ def run(self):
     # source files
     *glob.glob("src/**/*", recursive=True),
     *glob.glob("python/jupyter/src/**/*", recursive=True),
+    *glob.glob("python/chemiscope/**/*", recursive=True),
```

Comment:
this is only listing TS files that enter the java script build, and there should not be any of these in `python/chemiscope`. Adding this line will trigger a rebuild of the javascript code too often (any time one of the Python file is modified) even if this rebuild is not necessary.

```suggestion
```